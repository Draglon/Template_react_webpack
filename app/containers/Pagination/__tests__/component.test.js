import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import PaginationComponent from '../component';

describe('<PaginationComponent /> container', () => {
  let component;
  const props = {
    query: 'some string',
    currentPage: 10,
    totalPages: 10,
    getPage: jest.fn(),
  };

  it('render property', () => {
    component = shallow(<PaginationComponent {...props} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
