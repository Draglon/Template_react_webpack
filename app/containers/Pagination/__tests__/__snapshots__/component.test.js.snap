// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaginationComponent /> container render property 1`] = `
<Fragment>
  <Row
    className="top-padding"
    gutter={0}
    justify="center"
    type="flex"
  >
    <Col>
      <Pagination
        className="pagination"
        current={10}
        defaultCurrent={1}
        onChange={[Function]}
        total={10}
      />
    </Col>
  </Row>
</Fragment>
`;
